### ISSUE TITLE
Webui State extension sometimes not loading until restart

### ISSUE THREAD

**Issue by Dajova** [lllyasviel/stable-diffusion-webui-forge#911]
I'm not sure if it's the fault of the extension itself or with forge, but it seems that it has issues to sometimes restore the state (params, config etc.). Some times it works fine, sometimes it doesnt work at al. Not sure why, it doesn't show anything out of the ordinary that i can see...

**Comment by KyokaNyx** [comment]
Hi, I think the best option is to revert to the working version from June. Here’s the link to the discussion:
https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/849

**Comment by Dajova** [comment]
I think i see a pattern... I use 2 different shortcuts with different command lines, one for updating extensions and whatnot and one that skips everything (including environment) and only the one that skips environment checking is fine, the other one "forgets" the state it was in previously. I think forge bugged something in its own environment.

**Comment by Dajova** [comment]
Yes, it was reverted the next pull i made... plus, it seems to have been fixed, as both shortcuts seems to load my extensions fine now. 
Only thing now is that the resolutions seems to halve themselves after each restart for some reason (not reverting, exactly halve on each value)... if i exit a session on 1280x1024, the next start of the webui starts up at 640x512.
